diff --git a/build_parallel.ps1 b/build_parallel.ps1
index 1f8b962..75a5022 100644
--- a/build_parallel.ps1
+++ b/build_parallel.ps1
@@ -46,7 +46,7 @@ foreach ($f in $files) {
 
     $psi = [System.Diagnostics.ProcessStartInfo]::new()
     $psi.FileName = $CC
-    $psi.Arguments = "-I. -I$relDir $extraInc `"$($f.FullName)`" -Oz -lkernel32 $extraLibs -ffreestanding -o `"$OutDir\$name.exe`""
+    $psi.Arguments = "-I. -I$relDir $extraInc -Wall -Wextra -Wpedantic -fno-builtin `"$($f.FullName)`" -Oz -lkernel32 $extraLibs -ffreestanding -o `"$OutDir\$name.exe`""
     $psi.WorkingDirectory = $PWD.Path
     $psi.UseShellExecute = $false
     $psi.CreateNoWindow = $true
